natashasthilaire/Swampy-Sells
# User

### About
This PR allows a user to input their location upon registration so that it will show up in the navigation bar and later appear for each item on the homepage. 

### Test Instructions
1. Run `user` branch locally
2. Register with a specified location (Clyde and Rosalinda, I think you can just manually add a `location` attribute in mongoDB)
3. Ensure specified location shows up under destination

### Screenshots
![image](https://github.com/natashasthilaire/Swampy-Sells/assets/93110503/93219632-6a52-470f-9ba2-6a0bcd4b1fe5)

### Closes Tickets
[SS-45](https://swampysells.atlassian.net/browse/SS-45)

### Related Tickets
